packages/core-utils/src/watcher.ts

```diff
@@ -98,19 +105,23 @@ export class Watcher {
 
     // Message has yet to be relayed, poll until it is found
     return new Promise(async (resolve, reject) => {
-      layer.provider.on(filter, async (log: any) => {
+      const handleEvent = async (log: any) => {
```

Comment:
Assuming this was introduced by adding the failed message event type, and we just now caught this from now having tested failed messages.  If so, should a watcher patch be included in the changelog?